sdk/resourcemanager/azure-resourcemanager-containerservice/src/main/java/com/azure/resourcemanager/containerservice/implementation/KubernetesClusterImpl.java

```diff
@@ -117,6 +139,17 @@ public String version() {
         return new byte[0];
     }
 
+    @Override
+    public byte[] userKubeConfigContent(Format format) {
+        if (format == null) {
+            return userKubeConfigContent();
+        }
+        for (CredentialResult config : userKubeConfigs(format)) {
+            return config.value();
+        }
```

Comment:
A bit strange to only return the first. User can still get all if call `userKubeConfigs(format)`?

Reply:
Can't find any docs on this format.. The description says it supports list config by this format in AAD cluster. Currently we don't have convenient layer support for AAD profile so it's for clusters created in `portal` or `CLI`.

Reply:
So how does your test case works?

Reply:
It works against normal clusters and they seems to ignore this format... For AAD ones, it can only be done locally for now...

Reply:
For the same AAD cluster, 
for `kubeconfig` with format `AZURE`:
```yaml
users:
- name: clusterUser_rg-xiaofei_testxiaofeikubecluster
  user:
    auth-provider:
      config:
        apiserver-id: xxx
        client-id: xxx
        config-mode: "1"
        environment: AzurePublicCloud
        tenant-id: xxx
      name: azure
```

for format `EXEC`:
```yaml
users:
- name: clusterUser_rg-xiaofei_testxiaofeikubecluster
  user:
    exec:
      apiVersion: client.authentication.k8s.io/v1beta1
      args:
      - get-token
      - --environment
      - AzurePublicCloud
      - --server-id
      - xxx
      - --client-id
      - xxx
      - --tenant-id
      - xxx
      - --login
      - devicecode
      command: kubelogin
      env: null
```